airbyte-cdk/bulk/toolkits/load-iceberg-parquet/src/main/kotlin/io/airbyte/cdk/load/command/iceberg/parquet/IcebergCatalogSpecifications.kt

```diff
@@ -55,6 +55,20 @@ interface IcebergCatalogSpecifications {
     @get:JsonProperty("main_branch_name", defaultValue = "main")
     val mainBranchName: String
 
+    /**
+     * The name of the database to be used when building the Table identifier
+     *
+     * This database name will only be used if the stream namespace is null, meaning when the
+     * `Destination Namespace` setting for the connection is set to `Destination-defined` or
+     * `Source-defined`
+     */
+    @get:JsonSchemaTitle("Database Name")
+    @get:JsonPropertyDescription(
+        """The database name. This will ONLY be used if the `Destination Namespace` setting for the connection is set to `Destination-defined` or `Source-defined`"""
+    )
+    @get:JsonProperty("database_name")
+    val databaseName: String?
```

Comment:
we maybe should put this on the glue/nessie spec objects, instead of the top-level iceberg spec?

b/c then we can use catalog-specific terminology (glue "database", nessie "namespace", etc)

and also avoid having problems if a catalog doesn't support namespaces